offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -479,78 +481,110 @@ class OfferToPDFConverter implements OfferExporter {
 
         }
 
-        static String tableHeader(String elementId) {
-            //1. add pagebreak
-            //2. create empty table for elementId
+        static String pageBreak() {
```

Comment:
nice idea to move that to its own table +1

Reply:
🧠